### PR TITLE
feat: added github icon to the footer

### PR DESCRIPTION
**Pull request by rcmtcristian**
have to fix flex alignment, and change the color of the icon

### CODE REVIEW

**Review by coderbyheart — CHANGES_REQUESTED** [review]
Please use SVG for the logo.

**Comment by rcmtcristian** [comment]
changed the color of the github icon and margin for the footer

**Comment by rcmtcristian** [comment]
Missed that request and closed it but now it should be updated.

**Review by coderbyheart — CHANGES_REQUESTED** [review]
Please use a proper SVG, not a traced bitmap.

**Comment by coderbyheart** [comment]
> Please use a proper SVG, not a traced bitmap.

GitHub Logos can be found on https://github.com/logos

**Comment by jtfairbank** [comment]
Hey @coderbyheart I was looking at those with @rcmtcristian when we did our onboarding session.  We went with the .png for starters to focus on the code.

@rcmtcristian could you see if you could covert the `.eps` or `.ai` vector images available on the github logo download page to a `.svg`?  That seems like it'd resolve this issue!

**Comment by rcmtcristian** [comment]
> Hey @coderbyheart I was looking at those with @rcmtcristian when we did our onboarding session. We went with the .png for starters to focus on the code.
> 
> @rcmtcristian could you see if you could covert the `.eps` or `.ai` vector images available on the github logo download page to a `.svg`? That seems like it'd resolve this issue!

just converted them in like that, hopefully it was the correct way

File: src/components/social-media/SocialMediaLink.tsx [hunk 1]
```diff
@@ -10,7 +10,7 @@ interface Props {
 const SocialMediaLink: FC<Props> = ({ href, src, altText }) => {
   return (
     <ExternalLink href={href}>
-      <aside className="w-16 h-16 flex flex-col justify-center items-center">
+      <aside className="w-16 h-16 flex flex-col justify-center items-center -mr-4 ">
```
Comment (coderbyheart) [inline src/components/social-media/SocialMediaLink.tsx]:
```suggestion
      <aside className="w-16 h-16 flex flex-col justify-center items-center -mr-4">
```

**Review by coderbyheart — CHANGES_REQUESTED** [review]
Icon should be blue.

image

**Comment by jtfairbank** [comment]
@rcmtcristian you should be able to open the `.svg` file in a text editor and set the color to Distribute Aid's blue hex code `#09328B`.  You can look at the other social icon `.svg` files for how to do this using the `fill` attribute (`fill="#09328B"`).

**Comment by rcmtcristian** [comment]
super weird I made sure it had the correct color, and it showed it correct on my end. gonna recheck it

**Comment by rcmtcristian** [comment]
just saw that my end had not updated correctly after I changed the file, fixed now to how it should be. thank you for being so patient

**Comment by jtfairbank** [comment]
Hey @rcmtcristian no worries at all!  It's often easier to see these things as reviewers.  Last request: looks like there are some little black dots in the tail, can you try finding out where the color is set for those in the `.svg` and updating it to the DA blue as well?  Thanks!

**Comment by rcmtcristian** [comment]
dots in the svg changed to the correct color

**Comment by jtfairbank** [comment]
Thanks so much @rcmtcristian! :100: 

Merge when ready @coderbyheart, or def point out any other change requests! :smile:

**Review by hola-soy-milk — APPROVED** [review]
Just gave this a look, fantastic work @rcmtcristian!

**Comment by coderbyheart** [comment]
LGTM, however, did you notice that the new icon is included using an `` tag, while the others are data embed?

**Comment by hola-soy-milk** [comment]
> LGTM, however, did you notice that the new icon is included using an `` tag, while the others are data embed?

Aha! Could that be related to the source? Given it's a Github SVG?

**Comment by coderbyheart** [comment]
Ah, I see. The SVG includes multiple logos, however the SVG viewbox crops it.
Will fix it.

**Comment by hola-soy-milk** [comment]
Yay! Thank you all so much! 🎉

**Comment by rcmtcristian** [comment]
Awesome! thank you all so much for guiding me through

**Comment by coderbyheart** [comment]
Teamwork!

**Pull request merged**